PostFreeSurfer/PostFreesurferPipelineLongPrep.sh

```diff
@@ -0,0 +1,433 @@
+    fi
+else 
+    :
+fi
+#end block
+
+############################################################################################################
+# The next block creates original brain masks from wmparc.mgz. The final brain mask will be computed later.
+# Convert FreeSurfer Volumes
+FreeSurferFolder_TP_long="$StudyFolder/$Timepoint_long/T1w/$Timepoint_long"
+FreeSurferFolder_Template="$T1w_dir_template/$Template"
+T1wImageBrainMask_orig="$T1wImageBrainMask"_orig
+
+if (( TemplateProcessing == 0 )); then 
+    log_Msg "Timepoint $Timepoint_long: creating brain mask in template acpc_dc space"
+    mri_convert -rt nearest -rl "$T1w_dir_long/T1w_acpc_dc.nii.gz" "$FreeSurferFolder_TP_long"/mri/wmparc.mgz "$T1w_dir_long"/wmparc_1mm.nii.gz
+    applywarp --rel --interp=nn -i "$T1w_dir_long"/wmparc_1mm.nii.gz -r "$T1w_dir_long"/"${T1wImage}_acpc_dc".nii.gz --premat=$FSLDIR/etc/flirtsch/ident.mat -o "$T1w_dir_long"/wmparc.nii.gz
+
+    #Create FreeSurfer Brain Mask
+    fslmaths "$T1w_dir_long"/wmparc_1mm.nii.gz -bin -dilD -dilD -dilD -ero -ero "$T1w_dir_long"/"$T1wImageBrainMask_orig"_1mm.nii.gz
+    ${CARET7DIR}/wb_command -volume-fill-holes "$T1w_dir_long"/"$T1wImageBrainMask_orig"_1mm.nii.gz "$T1w_dir_long"/"$T1wImageBrainMask_orig"_1mm.nii.gz
+    fslmaths "$T1w_dir_long"/"$T1wImageBrainMask_orig"_1mm.nii.gz -bin "$T1w_dir_long"/"$T1wImageBrainMask_orig"_1mm.nii.gz
+    applywarp --rel --interp=nn -i "$T1w_dir_long"/"$T1wImageBrainMask_orig"_1mm.nii.gz -r "$T1w_dir_long"/"$T1wImage"_acpc_dc.nii.gz --premat=$FSLDIR/etc/flirtsch/ident.mat -o "$T1w_dir_long"/"$T1wImageBrainMask_orig".nii.gz
+else 
+    log_Msg "Template $Template: creating brain mask in template acpc_dc space"
+    mri_convert -rt nearest -rl "$T1w_dir_long/T1w_acpc_dc.nii.gz" "$FreeSurferFolder_Template"/mri/wmparc.mgz "$T1w_dir_template"/wmparc_1mm.nii.gz
+    applywarp --rel --interp=nn -i "$T1w_dir_template"/wmparc_1mm.nii.gz -r "$T1w_dir_long"/"${T1wImage}_acpc_dc".nii.gz --premat=$FSLDIR/etc/flirtsch/ident.mat -o "$T1w_dir_template"/wmparc.nii.gz
+    fslmaths "$T1w_dir_template"/wmparc_1mm.nii.gz -bin -dilD -dilD -dilD -ero -ero "$T1w_dir_template"/"$T1wImageBrainMask_orig"_1mm.nii.gz
+    ${CARET7DIR}/wb_command -volume-fill-holes "$T1w_dir_template"/"$T1wImageBrainMask_orig"_1mm.nii.gz "$T1w_dir_template"/"$T1wImageBrainMask_orig"_1mm.nii.gz
+    fslmaths "$T1w_dir_template"/"$T1wImageBrainMask_orig"_1mm.nii.gz -bin "$T1w_dir_template"/"$T1wImageBrainMask_orig"_1mm.nii.gz
+    applywarp --rel --interp=nn -i "$T1w_dir_template"/"$T1wImageBrainMask_orig"_1mm.nii.gz -r "$T1w_dir_long"/"$T1wImage"_acpc_dc.nii.gz --premat=$FSLDIR/etc/flirtsch/ident.mat -o "$T1w_dir_template"/"$T1wImageBrainMask_orig".nii.gz
+fi
+
+##############################################################################################################
+# The next block runs gain field correction in the template space for the TXw images.
```

Comment:
resamples? we tend to use "bias field" instead of "gain field" as the terminology in the HCP Pipelines.

Reply:
Ok